@kiva/kv-components/vue/KvCommentsListItem.vue

```diff
@@ -99,17 +112,34 @@ export default {
 		},
 	},
 	setup(props) {
+		const showInput = ref(false);
+		const commentsAddRef = ref(null);
+
 		const onClick = (reaction, value) => {
-			props.handleClick({
-				reaction,
-				id: props.comment?.id ?? null,
-				userId: props.comment?.user_id ?? null,
-				isChild: true,
-				value,
-			});
+			if (reaction === REPLY_COMMENT_EVENT) {
+				showInput.value = true;
+				nextTick(() => {
+					commentsAddRef.value.$refs.input.focus();
+				});
+			}
+
+			if (value) {
+				props.handleClick({
```

Comment:
Not sure why I didn't notice this before, but this should be an emitted event, rather than a prop method.

Reply:
Agreed! I can add this to the components tweaks as building a generic payload...

Reply:
@christian14b @roger-in-kiva Will this be a follow-up task?

Reply:
Yeah, I have this here: https://kiva.atlassian.net/browse/ACK-1027